src/raymath.h

```diff
@@ -114,6 +114,14 @@
         float z;
     } Vector3;
 
+    // Vector4 type
```

Comment:
Actually, Quaternion could also be changed in this case:
```c
// Vector4 type
typedef struct Vector4 {
    float x;
    float y;
    float z;
    float w;
} Vector4;

// Quaternion type, same as Vector4
typedef Vector4 Quaternion;
```

Reply:
Done